docs/network_discovery.md

```diff
@@ -77,6 +77,20 @@ entry.async_on_unload(
 )
 ```
 
+
+### Subscribing to unavailable callbacks
+
+To get a callback when the bluetooth stack can no longer see a device, call the `bluetooth.async_track_unavailable` API.
```

Comment:
```suggestion
To get a callback when the bluetooth stack can no longer see a device, call the `bluetooth.async_track_unavailable` API. For performance reasons, it may take up to five minutes to get a callback once the device is no longer seen.
```